tmt/base.py

```diff
@@ -2917,34 +2917,32 @@ def name_filter(nodes: Iterable[fmf.Tree]) -> list[fmf.Tree]:
             tests = [
                 Test(node=test, logger=self._logger.descend()) for test in self.tree.prune(
                     keys=keys, sources=cmd_line_names)]
-
-        else:
+        elif not unique and names:
```

Comment:
@martinhoyer something @happz is preparing a styling guide afaik, prefer a space before rather ...

Reply:
and the next space can be removed, as it was before

Reply:
Thanks. Added the space back.